config/locales/en.yml

```diff
@@ -1089,6 +1089,8 @@ en:
     partners_ucl: "UCL"
     partners_bytemark: "Bytemark Hosting"
     partners_partners: "partners"
+    intro_tou_html: "Use of this site is subject to our %{tou_link}." 
```

Comment:
Is this actually used anywhere?

Reply:
This was the text in the welcome box in the 1st version that @systemed didn't want, and that I then removed. Fixed now.